pkg/kafka/context.go

```diff
@@ -0,0 +1,59 @@
+package kafka
+
+import (
+	"context"
+)
+
+type contextKey int
+
+const (
+	_ contextKey = iota
+	topicContextKey
+	partitionContextKey
+	partitionOffsetContextKey
+)
+
+func setTopicToCtx(ctx context.Context, topic string) context.Context {
+	return context.WithValue(ctx, topicContextKey, topic)
+}
+
+// MessageTopicFromCtx returns kafka topic of consumed message
+func MessageTopicFromCtx(ctx context.Context) string {
```

Comment:
I like idea of adding topic to the ctx, but maybe we can add `MessageTopicFromCtx` to router here: https://github.com/ThreeDotsLabs/watermill/blob/master/message/router.go#L318 ? in that way it may work out of the box with all Pub/Subs :) 

cc @m110 @maclav3

Reply:
Good idea 👍 🎖